modules/ROOT/pages/database-administration/aliases/manage-aliases-standard-databases.adoc

```diff
@@ -401,20 +412,51 @@ To view the remote database alias details, use the `SHOW ALIASES FOR DATABASE` c
 .Query
 [source, cypher]
 ----
-SHOW ALIAS `remote-northwind`
+SHOW ALIAS `remote-northwind-stored-credentials`
 FOR DATABASE
 ----
 
 .Result
 [role="queryresult"]
 ----
-+----------------------------------------------------------------------------------------------------------+
-| name               | composite | database               | location | url                       | user    |
-+----------------------------------------------------------------------------------------------------------+
-| "remote-northwind" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "alice" |
-+----------------------------------------------------------------------------------------------------------+
++-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+| name                                  | composite | database               | location | url                       | credentials                 | user    |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+| "remote-northwind-stored-credentilas" | NULL      | "northwind-graph-2020" | "remote" | "neo4j+s://location:7687" | "STORED NATIVE CREDENTIALS" | "alice" |
++-----------------------------------------------------------------------------------------------------------------------------------------------------------+
+----
+
+The `OIDC CREDENTIAL FORWARDING` clause is used to configure a remote database alias to use the logged-in user's OIDC credentials to authenticate to the remote DBMS.
+In order to use this method to authenticate, the local DBMS and remote DBMS needs to have SSO authentication and authorization through identity providers implementing the OIDC standard configured, see the xref:authentication-authorization/sso-integration.adoc[SSO integration] on how to configure OIDC identity providers.
+
+.Query
+[source, cypher]
+----
+CREATE ALIAS `remote-northwind-oidc-credential-forwarding` FOR DATABASE `northwind-graph-2020`
+AT "neo4j+s://location:7687"
+OIDC CREDENTIAL FORWARDING
 ----
 
+Since the alias use the credentials of the logged-in user when targeting the remote DBMS, there are no stored credentials and user will be `NULL`.
```

Comment:
```suggestion
Since the alias uses the credentials of the logged-in user when targeting the remote DBMS, there are no stored credentials, and the user will be `NULL`.
```